templates/papers.html

```diff
@@ -12,6 +12,10 @@
 
 {% endblock %}
 
+{% block heading %}
+{{ components.section("Browse Papers") }}
+{% endblock %}
+
```

Comment:
i feel we don't need this title -- it took too much space

Reply:
The reason I added this was that some pages had titles whereas some did not. This leads to page content moving up and down when navigating. I can remove it if you still think the space is an issue.